File: WIMS-front/src/app/Observables/Role.ts
export class Role {
    constructor(private id: number) {}
    private roleName: string;

    get Id(): number {
        return this.id;
    }

    set Id(value: number) {
        this.id = value;
    }

    get RoleName(): string {
        return this.roleName;
    }

    set RoleName(value: string) {
        this.RoleName = value;
    }
}

File: back_end_fg/WebApplication1/WebApplication1/Models/Role.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WimsApiMKI.Models
{
    public class Role
    {
        private int _id;
        private string _roleName;

        public Role(int value)
        {
            this.Id = value;
        }

        #region GetSet
        public string RoleName
        {
            get { return _roleName; }
            set { _roleName = value; }
        }

        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }
        #endregion

    }
}
